docs/SUMMARY.md

```diff
@@ -52,31 +72,26 @@
   * [Command](cli/command.md)
   * [Clone Services](cli/clone-services.md)
   * [Shutdown](cli/shutdown.md)
-* [Hardware](hardware/README.md)
-  * [Supported Compute and Install](hardware/compute-install.md)
-  * [Kits and Sensors](hardware/kits-sensors.md)
+* [Communications](communications/README.md)
 * [Data](data/README.md)
```

Comment:
I don't think Data really belongs in this core section. Maybe there's a new section called "using nio" or something similar that includes data, UIs, and the CLI

Reply:
Building with nio